pkgs/development/libraries/rocfft/default.nix

```diff
@@ -19,13 +19,22 @@
 }:
 
 let
+  lib-zero = "librocfft-device-0.so.0.1";
+  lib-one = "librocfft-device-1.so.0.1";
+  lib-two = "librocfft-device-2.so.0.1";
+  lib-three = "librocfft-device-3.so.0.1";
+
   # This is over 3GB, to allow hydra caching we separate it
   rf = stdenv.mkDerivation (finalAttrs: {
     pname = "rocfft";
     version = "5.4.2";
 
     outputs = [
       "out"
+      "zero"
+      "one"
+      "two"
+      "three"
```

Comment:
We should probably prefix those with ``lib-``.

Reply:
They are that way intentionally to prevent variable contamination.

Reply:
I apologize. Sleep deprivation made me a bit dumb and stubborn.
It's fixed now.
In case it's relevant, bash does not allow hyphens in variable names and I don't think a hyphen is worth using a ``placeholder``.